plextraktsync/config.default.yml

```
@@ -70,6 +70,9 @@ sync:
   # trakt - Trakt ratings have priority. Existing Plex ratings are overwritten.
   # plex - Plex ratings have priority. Existing Trakt ratings are overwritten.
   rating_priority: plex
+  # watched_status and ratings of medias not actually in your Plex server can 
```

```suggestion
  # watched_status and ratings of media not actually in your Plex server can 
```

English please: media is already plural